core/story-api/src/main/java/org/lgna/story/implementation/PerspectiveCameraMarkerImp.java

```diff
@@ -304,6 +304,10 @@ public Visual[] getSgVisuals() {
     return isVrActive ? sgVrVisuals : sgCameraVisuals;
   }
 
```

Comment:
i have no idea what putInstance does.  might warrant a comment?

Reply:
It is a scary subsystem on every Element that I want to remove, but today it is useful.
A comment has been added to the code too.